### PR TITLE
added reference to mssql exporter

### PR DESCRIPTION
**Pull request by awaragi**
Reference to a new exporter for Microsoft SQL Server.

### CODE REVIEW

File: content/docs/instrumenting/exporters.md [hunk 1]
```diff
@@ -37,6 +37,7 @@ wide variety of JVM-based applications, for example [Kafka](http://kafka.apache.
    * [ElasticSearch exporter](https://github.com/justwatchcom/elasticsearch_exporter)
    * [Memcached exporter](https://github.com/prometheus/memcached_exporter) (**official**)
    * [MongoDB exporter](https://github.com/dcu/mongodb_exporter)
+   * [Microsoft SQL server exporter](https://github.com/awaragi/prometheus-mssql-exporter)
```
Comment (brian-brazil) [inline content/docs/instrumenting/exporters.md]: MSSQL is the more common name.

Keep the list sorted.

**Comment by awaragi** [comment]
that's a fair recommendation! I will rework the logic to be on demand only.

**Comment by awaragi** [comment]
I've released a new version of the exporter which only fetches data upon scrape. Thanks @brian-brazil for your suggestion.

**Comment by brian-brazil** [comment]
Thanks!

**Pull request merged**